### ISSUE TITLE
Doesn't work with restify 5 

### ISSUE THREAD

**Issue by alimousazy** [TabDigital/restify-cors-middleware#39]

**Comment by simonepri** [comment]
Could you give use more information? Doesn't work is really vague.

**Comment by alimousazy** [comment]
I upgraded to restify 5 and it stop working here is the error output

`Unhandled rejection TypeError: Cannot read property 'forEach' of undefined
    at module.exports (/Users/aalrahahlah/projects/homes-xander-orch/node_modules/restify-cors-middleware/src/index.js:39:18)
`

**Comment by simonepri** [comment]
Did you install the latest version of the package? Which version are you using?

**Comment by alimousazy** [comment]
I'm using 1.0

**Comment by simonepri** [comment]
Can you post the lines where you are using the middleware? May help others to fix it if is a bug.

**Comment by chq-matteo** [comment]
https://github.com/restify/node-restify/blob/5.x-docs/4TO5GUIDE.md#cors

**Comment by simonepri** [comment]
@alimousazy any news about it?

**Comment by joaoritter** [comment]
+1

**Comment by alexgoldstone** [comment]
It seems to work fine for me following the example in the README but I do get this related warning from NPM:
```
npm WARN restify-cors-middleware@1.0.0 requires a peer of restify@2.6.x - 4.3.x but none was installed.
```

**Comment by simonepri** [comment]
@alexgoldstone this PR should fix it as soon as we merge it
https://github.com/TabDigital/restify-cors-middleware/pull/38
@rprieto could you see it?

**Comment by simonepri** [comment]
@alexgoldstone that was fixed in `1.0.1`

**Comment by rprieto** [comment]
Thanks for your patience! Let us know if you have any other issues.
